Source/WebKit/UIProcess/RemoteLayerTree/RemoteAnimation.cpp

```diff
@@ -47,7 +47,7 @@ RemoteAnimation::RemoteAnimation(const WebCore::AcceleratedEffect& effect, const
 
 void RemoteAnimation::apply(WebCore::AcceleratedEffectValues& values)
 {
-    Ref { m_effect }->apply(values, m_timeline->currentTime(), m_timeline->duration());
+    m_effect->apply(values, m_timeline->currentTime(), m_timeline->duration());
```

Comment:
Shouldn't / can't we make `m_effect` const?


> Normally, we require having a smart pointer on the stack before calling a non-trivial member function on data members, even though that data members already are smart pointers. The reasoning is that those data members may get reassigned in the middle of the call, which could lead to use-after-free. However, if the data member is never reassigned and is a smart pointer, having a smart pointer on the stack is actually unnecessary

https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines#mark-data-members-that-are-smart-pointers-as-const-if-they-never-get-reassigned